python/cudf/cudf/tests/test_orc.py

```diff
@@ -1913,3 +1913,15 @@ def test_orc_reader_empty_decomp_data(datadir):
     got = cudf.read_orc(path)
 
     assert_eq(expect, got)
+
+
+# Test the case where top level struct has nulls, but the nested struct is not
+# nullable. In this case there is no data in the second level, but we still
+# need to pass the parent null mask to the third level
+def test_orc_reader_empty_deeply_nested_level(datadir):
```

Comment:
Usually we put the test description inside the `def`. Also I rewrapped the lines for you in this suggestion. 👍 
```suggestion
def test_orc_reader_empty_deeply_nested_level(datadir):
    # Test the case where top level struct has nulls, but the nested struct is
    # not nullable. In this case there is no data in the second level, but we
    # still need to pass the parent null mask to the third level.
```